build/gen_stub.php

```diff
@@ -342,22 +423,52 @@ public function getSendByString(): string {
         throw new Exception("Invalid sendBy value");
     }
 
-    public function hasDefaultValue(): bool {
+    public function getMethodsynopsisType(): Type {
```

Comment:
Personally I'd spell it MethodSynopsis here and elsewhere.

Reply:
Ye, makes sense for me!